apps/vr-tests-react-components/src/stories/Accordion.stories.tsx

```diff
@@ -0,0 +1,161 @@
+import { storiesOf } from '@storybook/react';
```

Comment:
Contents of all story files are exactly the same. I've just changed the naming to removed the `Converged` suffix to each of the files